pkg/sanity/controller.go

```diff
@@ -193,8 +193,73 @@ var _ = Describe("ListVolumes [Controller Server]", func() {
 		Expect(serverError.Code()).To(Equal(codes.Aborted))
 	})
 
-	// TODO: Add test which checks list of volume is there when created,
-	//       and not there when deleted.
+	It("check the presence of new volumes in the volume list", func() {
```

Comment:
>  // TODO: Add test which checks list of volume is there when created, 
>  //       and not there when deleted. 

It wasn't clear what this TODO was actually about, but I felt it's about adding and removing, and checking volume list. Need a little verification.